Gemfile

```diff
@@ -81,28 +77,21 @@ gem 'active_link_to'
 gem 'bootsnap', require: false
 gem 'enumerize'
 gem 'friendly_id'
-gem 'jbuilder'
-# gem 'listen' # needed?
```

Comment:
we've added this back in? is that on purpose?

Reply:
This is needed by the asset pipeline for watching when files have changed.